Cargo.toml

```diff
@@ -13,7 +13,7 @@ repository = "https://github.com/Synphonyte/leptos-struct-table"
 
 [dependencies]
 leptos = { version = "0.7.0" }
-leptos-struct-table-macro = { version = "0.11.2" }
+leptos-struct-table-macro = { version = "0.12", git = "https://github.com/kstep/leptos-struct-table-macro" branch = "leptos-0.7" }
```

Comment:
This line must be updated once `leptos-struct-table-macro` is updated and published.